crates/gitbutler-hunk-dependency/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub mod diff;
```

Comment:
There is still quite a bit of panic-y access by index. You can see it by adding this clippy rule at the top of the file
```
#![warn(clippy::indexing_slicing)]
```
While annoying to replace now, it will pay dividends if/when we hit stuff to debug